Makefile

```diff
@@ -49,6 +49,9 @@ python-quality-test:
 
 run-tests: python-test python-quality-test
 
+run-integration-tests: install-dev-dependencies
```

Comment:
@BryanttV, I see that you've moved all the tests into their respective unit folders. I'm guessing we're not using something like `pytest ./eox_core --ignore-glob='**/unit/*'` because the integration tests for the backend imports would fail, right?

Reply:
Yes, and also for the tests inside `eox_core/tests`. But what do you think about this change? We would not need to modify the command as long as the new tests are inside an `integration` folder.

https://github.com/eduNEXT/eox-core/commit/74a04f5358c97c11afb7b3d48aa184de36eea654

Reply:
@BryanttV nice, I agree with those changes. We can leave the `--ignore-glob='**/edxapp_wrapper/*'` while we figure out what to do with the edxapp_wrapper test